src/main/kotlin/io/snyk/plugin/cli/ConsoleCommandRunner.kt

```diff
@@ -98,6 +99,10 @@ open class ConsoleCommandRunner {
         commandLine.environment["SNYK_INTEGRATION_VERSION"] = pluginInfo.integrationVersion
         commandLine.environment["SNYK_INTEGRATION_ENVIRONMENT"] = pluginInfo.integrationEnvironment
         commandLine.environment["SNYK_INTEGRATION_ENVIRONMENT_VERSION"] = pluginInfo.integrationEnvironmentVersion
+        val proxySettings = HttpConfigurable.getInstance()
+        if (proxySettings != null && proxySettings.USE_HTTP_PROXY && proxySettings.PROXY_HOST.isNotEmpty()) {
+            commandLine.environment["https_proxy"] = "http://${proxySettings.PROXY_HOST}:${proxySettings.PROXY_PORT}"
```

Comment:
Why `http` hardcoded? What about respecting auth parameters?

Reply:
`http` is somehow needed by the CLI - I don't know why they expect a hard-coded http even for https. Try it out, you'll see an exception. About authentication: yes, it should be part of the hostname, I agree.